libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription_container.c

```diff
@@ -283,7 +283,7 @@ bool IotMqtt_RemoveSubscription( _mqttSubscription_t * pSubscriptionArray,
 /*-----------------------------------------------------------*/
 
 void IotMqtt_RemoveAllMatches( _mqttSubscription_t * pSubscriptionArray,
-                               _packetMatchParams_t * pMatch )
+                               const _packetMatchParams_t * pMatch )
```

Comment:
same comment about const qualifier

Reply:
I have added the const qualifier as the matching params should not change.